docs/AOTA.md

```diff
@@ -0,0 +1,43 @@
+# AOTA Updates
+
+## Supported AOTA commands and their functionality:
+
+### Supported ***docker*** commands
```

Comment:
I've found the docker AOTA commands to be the most confusing ones. An example with (load or import or pull) + (list) + (remove) would be good, also explaining how image version numbers and rollback work.

Reply:
@gblewis1 - There are examples in the Azure and Thingsboard guides.  This document is for them to reference these charts so that it's not duplicated in 3 different documents.  We can expand on that in another PR.  I'll put it  in the follow-up story of things that aren't going to make it into this PR.